web/src/routes/Events.jsx

```diff
@@ -495,9 +501,15 @@ export default function Events({ path, ...props }) {
                   {viewEvent !== event.id ? null : (
                     <div className="space-y-4">
                       <div className="mx-auto max-w-7xl">
-                        {event.has_clip ? (
-                          <>
-                            <Heading size="lg">Clip</Heading>
+                        <div className='flex justify-center w-full py-2'>
+                          <Tabs selectedIndex={event.has_clip && eventDetailType == 'clip' ? 0 : 1} onChange={handleEventDetailTabChange} className='justify'>
+                            <TextTab text='Clip' disabled={!event.has_clip} />
+                            <TextTab text={event.has_snapshot ? 'Best Image' : 'Thumbnail'} />
```

Comment:
You alluded to it, but I think we definitely want this called `Snapshot`. It can be confusing where it comes from and also if downloading the snapshot that is the exact image that will be downloaded.

Reply:
Done!